ocp_install_env.sh

```diff
@@ -130,6 +130,22 @@ EOF
   fi
 }
 
+function apiVIP_ingressVIP_dualstack() {
+  # dual stack will be added 4.12, let's keep compat mode
+  # The char , is used for multiple vips which is supported only in dualstack mode
+  if [[ "${API_VIP}" == *","* ]]; then
```

Comment:
This makes a couple of assumptions I'm not sure are valid:

- That if there are multiple API VIPs there will always be multiple ingress VIPs
- That we only want to use the new fields if multiple VIPs are specified

I think this needs to be based on the openshift version like it is for dnsvip, and the logic should handle a mismatch in the number of VIPs between ingress and API. I'm not sure the latter is strictly necessary, but there's no inherent reason you couldn't have two API VIPs and only one ingress (or vice versa).